src/plugin.ts

```diff
@@ -15,10 +15,15 @@ type PluginConfig = {
    * causing conflicts.
    */
   iife?: boolean;
+  /**
+   * Filter the output css content. This can be required as it is possible that packages
+   * which generate css include extra characters or metadata that is not desired.
+   */
+  filterOutput?: (out: string) => string
```

Comment:
I’d rather have this named `transformOutput`, as it’s transforming source rather than filtering it.